### ISSUE TITLE
Not possible to add non-english reflection

### ISSUE THREAD

**Issue by igorlesnenko** [ParabolInc/parabol#7987]
- create a retro
- try to add non-english reflection, e.g. "тест"
- see reflection not added
- see error in logs

```
15:02:18 1|GraphQL Executor  | TypeError: Cannot read properties of undefined (reading 'score')
```

**Comment by igorlesnenko** [comment]
Sentry issue: [ACTION-PRODUCTION-E76](https://parabol.sentry.io/issues/4057751322/?referrer=github_plugin)

**Comment by Dschoordsch** [comment]
Not all non-English language seem affected, German works fine. Only "team" + "enterprise" organizations should be affected.

**Comment by tianrunhe** [comment]
There's a bug on error handling [here](https://github.com/ParabolInc/parabol/blob/master/packages/server/graphql/mutations/helpers/manageGoogleNLPErrorResponse.ts#L6). When the language is not supported, the response from Google looks like this:

```
{
  error: {
    code: 400,
    message: 'The language ru is not supported for document_sentiment analysis.',
    status: 'INVALID_ARGUMENT'
  }
}
```
There's no array in the outer block.

**Comment by igorlesnenko** [comment]
Fixed by #7988
